config/locales/cy.yml

```diff
@@ -86,3 +86,5 @@ cy:
           pension_calculator_html: Defnyddiwch ein <a href="#">Cyfrifiannell Pensiwn</a> i weld faint o bensiwn y bydd eich cronfa yn ei chronni dros amser.
           workplace_pensions_html: Dysgwch ragor am <a href="#">bensiynau gweithle</a>.
           budget_planner_html: Defnyddiwch ein <a href="#">cynllunydd cyllideb</a> i weld pa effaith y bydd eich cyfraniadau yn eu cael ar eich incwm.
+      edit_frequency:
+        label: Change salary frequency
```

Comment:
I don't know if I like the idea of adding TODO comments on the code. We can add a note on the card or any documentation but on the code, I tend to disagree because will be one more source of truth to see all todo in the project.